### PR TITLE
fixes token formatting portfolio

### PR DESCRIPTION
**Pull request by rudolfix**
several small improvements: please see commit list

### CODE REVIEW

File: packages/web/app/components/eto/overview/EtoOverviewThumbnail/EtoStatusManager.tsx [hunk 1]
```diff
@@ -139,6 +139,9 @@ const EtoStatusManager = ({ eto }: IExternalProps) => {
       );
     }
 
+    case EEtoState.SUSPENDED:
+      return <></>;
```
Comment (Moejoe90) [inline packages/web/app/components/eto/overview/EtoOverviewThumbnail/EtoStatusManager.tsx]: We need a ticket for this probably

File: packages/web/app/components/issuer-dashboard/EtoDashboard.tsx [hunk 2]
```diff
@@ -155,6 +155,7 @@ const EtoDashboardStateViewComponent: React.FunctionComponent<IEtoStateRender> =
         </>
       );
     case EEtoState.PENDING:
+    case EEtoState.SUSPENDED:
```
Comment (Moejoe90) [inline packages/web/app/components/issuer-dashboard/EtoDashboard.tsx]: So `SUSPENDED` is just like `PENDING` from the issuer side really?

Comment (rudolfix) [reply packages/web/app/components/issuer-dashboard/EtoDashboard.tsx]: no - linter does not allow for code duplications!!

SUPSENDED is a dead end where I had to remove ETO from Universe contract. it's dead. and we do not need to cater to this too much

Comment (rudolfix) [reply packages/web/app/components/issuer-dashboard/EtoDashboard.tsx]: I will explain what suspended is on standup tomorrow quickly so you have all the info

Comment (glupiawymowka) [reply packages/web/app/components/issuer-dashboard/EtoDashboard.tsx]: we will need a ticket for this indeed, cause this state is causing minor errors, but currently, this is an internal state anyway (and let's hope we don't have to suspend any company soon 😅 )

**Pull request merged**